backend/capellacollab/settings/modelsources/t4c/license_server/routes.py

```diff
@@ -95,11 +95,33 @@ def delete_t4c_license_server(
     crud.delete_t4c_license_server(db, license_server)
 
 
+@router.get(
```

Comment:
I'd not add another route. We already have the `get_t4c_license_servers` route, which doesn't contain much more valuable information. Just change the `admin_route` to `router` and make use of the existing route.